code/game/objects/items/pneumaticCannon.dm

```diff
@@ -337,6 +337,57 @@
 	maxWeightClass = 6 //2 pies
 	charge_ticks = 2 //4 second/pie
 
+/obj/item/pneumatic_cannon/speargun
```

Comment:
This already exists and was changed to a bolt-action rifle. It currently is only located in a specific traitor bundle.

Reply:
KIT_WHITE_WHALE_HOLY_GRAIL

Reply:
Ah, i found it. Will work on a fix.